### ISSUE TITLE
Automatically sync prior to install, if there is no local repo info

### ISSUE THREAD

**Issue by justinethier** [cyclone-scheme/cyclops#3]

**Comment by arthurmaciel** [comment]
Shoulnd't cyclops sync prior to install on every ocasion, even when there is a local repo info?

**Comment by justinethier** [comment]
A sync is (most likely) not required if the repo was updated recently though. Maybe there could be a time limit?

Other package managers (pacman, yum?) will alert if a "sync" has not occurred for a long time and may fail to find/install a package if they are out of date.

**Comment by arthurmaciel** [comment]
I thought every time a package was created in cyclone-packages, this repo should create a new index.dat file and cyclops should update it. Is there a different mechanism for updating index.dat that I'm not aware of?

**Comment by justinethier** [comment]
No, you're right. I was thinking about how this would scale up, but maybe that doesn't matter so much at first, when there are relatively few packages.
